workers/unity/Packages/io.improbable.gdk.core/Tests/Editmode/Connection/LocatorFlowTests.cs

```diff
@@ -54,6 +54,7 @@ public void LocatorFlow_dev_auth_fails_if_no_deployment_running()
             Assert.IsInstanceOf<AuthenticationFailedException>(aggregateException.InnerExceptions[0]);
         }
 
+#if !UNITY_EDITOR_OSX
```

Comment:
What's the reason we're only disabling this on OSX and not other platforms?

Reply:
These tests flake a fair bit on Mac. It's an extension of this PR: https://github.com/spatialos/gdk-for-unity/pull/1361

Reply:
Mind putting a comment there so it's easy to tell? Doesn't have to link to the PR, but just knowing why would be nice.

Reply:
Sure, will add